docs/content/preview/drivers-orms/nodejs/typeorm.md

```diff
@@ -0,0 +1,188 @@
+---
+title: TypeORM
+headerTitle: Use an ORM
+linkTitle: Use an ORM
+description: Node.js TypeORM support for YugabyteDB
+headcontent: Node.js ORM support for YugabyteDB
+aliases:
+  - /integrations/typeorm/
+menu:
+  preview:
+    identifier: typeorm-1
+    parent: nodejs-drivers
+    weight: 600
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="../sequelize/" class="nav-link ">
+      <i class="fa-brands fa-node-js" aria-hidden="true"></i>
+      Sequelize
+    </a>
+  </li>
+  <li >
+    <a href="../prisma/" class="nav-link ">
+      <i class="fa-brands fa-node-js" aria-hidden="true"></i>
+      Prisma
+    </a>
+  </li>
+  <li >
+    <a href="../typeorm/" class="nav-link active">
+      <i class="fa-brands fa-node-js" aria-hidden="true"></i>
+      TypeORM
+    </a>
+  </li>
+</ul>
+
+[TypeORM](https://typeorm.io/) is an ORM that can run in NodeJS, Browser, Cordova, PhoneGap, Ionic, React Native, NativeScript, Expo, and Electron platforms and can be used with TypeScript and JavaScript (ES2021). It supports both Active Record and Data Mapper patterns, unlike all other JavaScript ORMs currently in existence, which means you can write high-quality, loosely coupled, scalable, maintainable applications in the most productive way.
+
+Because YugabyteDB is PostgreSQL-compatible, TypeORM supports the YugabyteDB YSQL API.
+
+This page provides details for getting started with TypeORM for connecting to YugabyteDB.
+
+## Working with domain objects
```

Comment:
Not sure what the point of this section is. It doesn't have any content, just a description of what the section is for